_posts/2016-06-28-tips-for-upgrading-rails-3-2-to-4.markdown

```diff
@@ -0,0 +1,90 @@
+---
+layout: post
+title: "Tips for upgrading from Rails 3.2 to 4.0"
+date: 2016-06-28 11:08:00
+categories: ["rails"]
+author: "mauro-oto"
+---
+
+There are already quite a few guides in the wild to help with the upgrade of
+Rails 3.2 to Rails 4.0.
+The [official Rails guide](http://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html#upgrading-from-rails-3-2-to-rails-4-0)
+for upgrading from Rails 3.2 to 4.0 is very thorough.
+With the upcoming release of Rails 5.0, apps currently in production running
+Rails 3.2 should probably be updated to any stable Rails 4 release as soon as
+possible.
+
+Even an e-book on [upgrading Rails 3 to 4](https://github.com/alindeman/upgradingtorails4)
```

Comment:
@mauro-oto This sentence sounds weird. I understand the point, but it would be much easier to read if you changed the order of the words: "There is even an e-book about upgrading Rails 3 to 4, which ... " - Then the "which" ends up right next to the object that it's describing. 

A general tip is to try changing the order of the sentence components to see if it's easier to understand the meaning of the sentence and if it sounds better.